packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -232,12 +232,28 @@ class CameraValue {
 /// To show the camera preview on the screen use a [CameraPreview] widget.
 class CameraController extends ValueNotifier<CameraValue> {
   /// Creates a new camera controller in an uninitialized state.
+  ///
+  /// Deprecated. Please use [withSettings].
   CameraController(
     CameraDescription description,
-    this.resolutionPreset, {
-    this.enableAudio = true,
+    ResolutionPreset resolutionPreset, {
+    bool enableAudio = true,
+    this.imageFormatGroup,
+  })  : mediaSettings = MediaSettings(
+            resolutionPreset: resolutionPreset, enableAudio: enableAudio),
+        super(CameraValue.uninitialized(description));
```

Comment:
If I remember correctly, you should be able to use the `this.withSettings` passthrough.
```dart
CameraController(
    CameraDescription description,
    this.resolutionPreset, {
    this.enableAudio = true,
    ResolutionPreset resolutionPreset, {
    bool enableAudio = true,
    this.imageFormatGroup,
  })  : this.withSettings(description, mediaSettings: MediaSettings(
            resolutionPreset: resolutionPreset, enableAudio: enableAudio), imageFormatGroup: imageFormatGroup);
```

Reply:
This hasn't been addressed.